### ISSUE TITLE
Instaloader fails to fetch comments from hashtags

### ISSUE THREAD

**Issue by AriaShishegaran** [Thammus/instaloader#39]
started testing with fetching comments for hashtags and faced the error below always ending after the first comment being processed. test both on English and Persian with the same error.
```
root@lucifer:~# instaloader --comments "#fourtyfour"
Retrieving pictures with hashtag #fourtyfour
[  1] #fourtyfour #fourtyfour/2017-08-19_18-30-52.jpg [Who doesn't love summer? NO O…] Traceback (most recent call last):
  File "/usr/local/bin/instaloader", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 1384, in main
    args.profile_pic_only, args.fast_update, args.stories, args.stories_only, args.only_if)
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 1200, in main
    filter_func=filter_func)
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 1008, in download_hashtag
    downloaded = self.download_post(post, target='#' + hashtag)
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 805, in download_post
    self.update_comments(filename, post)
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 599, in update_comments
    comments.extend(post.get_comments())
  File "/usr/local/lib/python3.5/dist-packages/instaloader.py", line 301, in get_comments
    if self.comments == len(comments_in_metadata['edges']):
KeyError: 'edges'

```

**Comment by aandergr** [comment]
Thanks! This is fixed in 6300c21.

**Comment by AriaShishegaran** [comment]
should i be able to a use this update rightaway from ```v3-dev```? if yes this is not working.
i have installed ```v3-dev``` this way.
```pip3 install https://github.com/Thammus/instaloader/archive/v3-dev.zip```
my version of instaloader is 3rc0
after update it again returns the same error showed above...
